scripts/cli2md.rb

```diff
@@ -1,66 +1,94 @@
-require 'cgi'
+# frozen_string_literal: true
 
-incode = false
+require "cgi"
 
-first_param = false
+STATE_INITIAL = :inital
+STATE_CODE = :code
+STATE_TABLE = :table
 
+state = STATE_INITIAL
+
+# read from stdin or files in the args
 ARGF.each_with_index do |line, line_num|
-    # Headings
-    if /^(\w*):/ =~ line
-        puts "### #{$1}"
-    # Initial usage command
-    elsif line_num == 2
-        puts "`#{line.strip}`"
-    # Code sections
-    elsif /\s{3}\$/ =~ line
-        # Break code lines that end in \
-        if line [-2] == "\\"
-            incode=true
-        end
-        puts " #{line}"
-    # If previous line ends in \ indent to code block
-    elsif incode == true
-        incode = false
-        puts "   #{line}"
-    # Lists of parameters
-    #    --config value                         Path to a configuration file [$BUILDKITE_AGENT_CONFIG]
-    elsif /\s{3}(-{2}[a-z0-9\- ]*)([A-Z].*)$/ =~ line
-        if(first_param==false)
-            puts "<!-- vale off -->\n\n<table class=\"Docs__attribute__table\">"
-            first_param = true
-        end
-        command_and_value = $1.rstrip
-        command = command_and_value.split[0][2..-1]
-        value = command_and_value.split[1]
-        desc    = $2
+  # Replace all prime symbols with backticks. We use prime symbols instead of backticks in CLI
+  # helptext because Go does not support escaping backticks in backtick delimited strings.
+  # See: https://github.com/buildkite/agent/blob/main/clicommand/prime-signs.md
+  line.tr!("′", "`")
```

Comment:
Woah, my browser is not rendering this very well, but you be able to copy this into your text editor and see that is prime sign.